### PR TITLE
Add alert_creation to service resource

### PR DESCRIPTION
**Pull request by jtsaito**
[Trello card](https://trello.com/c/egabMaE4/534-1-allow-to-create-pagerduty-services-with-mergeable-alerts-incidents)

### CODE REVIEW

File: pagerduty/resource_pagerduty_service.go [hunk 1]
```diff
@@ -240,14 +253,15 @@ func resourcePagerDutyServiceRead(d *schema.ResourceData, meta interface{}) erro
...
+	d.Set("auto_resolve_timeout", service.AutoResolveTimeout)
 	d.Set("created_at", service.CreatedAt)
-	d.Set("escalation_policy", service.EscalationPolicy.ID)
 	d.Set("description", service.Description)
-	d.Set("auto_resolve_timeout", service.AutoResolveTimeout)
+	d.Set("escalation_policy", service.EscalationPolicy.ID)
 	d.Set("last_incident_timestamp", service.LastIncidentTimestamp)
-	d.Set("acknowledgement_timeout", service.AcknowledgementTimeout)
+	d.Set("name", service.Name)
+	d.Set("status", service.Status)
```
Comment (jansiwy) [inline pagerduty/resource_pagerduty_service.go]: shall we do styling changes which will increase the chance for merge conflicts?

File: website/docs/r/service.html.markdown [hunk 2]
```diff
@@ -52,6 +53,7 @@ The following arguments are supported:
   * `auto_resolve_timeout` - (Optional) Time in seconds that an incident is automatically resolved if left open for that long. Disabled if not set.
   * `acknowledgement_timeout` - (Optional) Time in seconds that an incident changes to the Triggered State after being Acknowledged. Disabled if not set.
   * `escalation_policy` - (Required) The escalation policy used by this service.
+  * `alert_creation` - (Optional) Must be one of two values. PagerDuty receives events from your monitoring systems and can then create incidents in different ways. Value "create_incidents" is default: events will create an incident that cannot be merged. Value "create_alerts_and_incidents" is the alternative: events will create an incident that cannot be merged.
```
Comment (jansiwy) [inline website/docs/r/service.html.markdown]: `create_alerts_and_incidents`: "... can be merged."

File: pagerduty/resource_pagerduty_service.go (hunk 1 above)
Comment (jtsaito) [reply pagerduty/resource_pagerduty_service.go]: @jansiwy not sure. i don't mind rolling back the ordering.

Comment (jansiwy) [reply pagerduty/resource_pagerduty_service.go]: @jtsaito i'm not sure either :-)

**Comment by jtsaito** [comment]
@jansiwy @fmasuhr Thanks!

**Pull request merged**